tests/commented.feature

```diff
@@ -25,52 +25,4 @@
 #         Examples: Fruit
 #             | thing  | other thing |
 #             | apple  | apple sauce |
-#             | banana | smoothie    |
-
-#     Scenario: Text and Table Scenario
-
-#         Test the additional options for a scenario
-
-#         Given step text
-#           '''
-#           Lorum ipsum solor sit amet.
-#           '''
-#         When the suite reaches a scenario table
-#             | name      | department  |
-#             | Barry     | Beer Cans   |
-#             | Pudey     | Silly Walks |
-#             | Two-Lumps | Silly Walks |
-#         Then the file is converted into rST
-
-#     Scenario: Normal scenarios with add on steps (And, But)
-
-#         Given something
-#         And something else
-#         When something happens
-#         Then something happened
-#         And something else also happened
-#         And another thing happened
-
-#     Scenario: Indentation is ignored when any step in the scenario has text or a table
-
-#         Given a step with some text
-#           '''
-#           Here be that said text!
-#           '''
-#         And an And step with some text too
-#           '''
-#           Hello again!
-#           '''
-#         And how about a table in there too
-#             | position | name         |
-#             | first    | Who          |
-#             | second   | What         |
-#             | third    | I don't know |
-#         When the fantasy has ended
-#         And all the children are gone
-#         Then something deep inside me
-#           '''
-#           Helps me to carry on
-#           '''
-#         And Encarnaciooooon
-#         And doodle-doodle-doodle-doo
+#             | banana | smoothie    |
```

Comment:
Looks like this line is missing it's newline: 
![Screen Shot 2021-12-13 at 11 44 33 AM](https://user-images.githubusercontent.com/240836/145853231-352f1c22-6e5f-40ad-90e1-1ac2b7bb3ad4.png)